### PR TITLE
remove unnecessary imports

### PR DESCRIPTION
**Pull request by finozzifa**
# Closes # (if applicable).

## Changes proposed in this Pull Request

The purpose of this PR is to remove all the imports present in the scripts/ folder that are not actually in use

## Checklist

- [x ] I consent to the release of this PR's code under the AGPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [x ] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [ ] Newly introduced dependencies are added to `envs/environment.yaml` and `doc/requirements.txt`.
- [ ] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [ ] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [ ] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/retrieve_databundle_light.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
     tutorial:  # configuration stating whether the tutorial is needed
 
 
-.. seealso::
+.. see also::
```
Comment (davide-f) [inline scripts/retrieve_databundle_light.py]: This is a keyword, can you revert?

File: scripts/build_powerplants.py [hunk 2]
```diff
@@ -115,9 +115,7 @@
     to_csv_nafix,
     two_digits_2_name_country,
 )
-from build_shapes import get_GADM_layer
```
Comment (davide-f) [inline scripts/build_powerplants.py:118]: In this script, the import logging should be dropped as well.
there are two lines (368 and 372) that still do logging.warning, but they should be revised into "logger.warning". Could you revise that and drop logging dependency?

**Review by davide-f — CHANGES_REQUESTED** [review]
Thanks @finozzifa :D
That's a very nice PR and soon ready to merge.

As additional element, the "import logging" should in theory be dropped everywhere except the helpers.
To me it seems that remains in build_powerplants for an old missing dependencies (see comment into the file) and also into build_cutout because we have an old "logging.info/warning" that should be revised into "logger.info/warning"
Could you check that?
After that, the PR can be merged :D

File: scripts/retrieve_databundle_light.py (hunk 1 above)
Comment (finozzifa) [reply scripts/retrieve_databundle_light.py]: Hi Davide. Thanks :)

File: scripts/build_powerplants.py (hunk 2 above)
Comment (finozzifa) [reply scripts/build_powerplants.py:118]: done!

**Review by davide-f — APPROVED** [review]
Great @finozzifa :D
This PR is ready to go :)

**Pull request merged**